cernopendata/modules/fixtures/data/docs/opera-news-first-release/opera-news-first-release.json

```diff
@@ -18,7 +18,7 @@
     "short_description": {
       "content": "As of today, the OPERA Collaboration has released the first set of its data samples for the Open Data Portal."
     },
-    "slug": "opera-news-first-release-2017",
+    "slug": "opera-news-first-release-2018",
```

Comment:
Good.  Can you please also update the `created` field to something like `2018-04-30`?